recipes/zstd/all/test_package/conanfile.py

```diff
@@ -15,13 +19,28 @@ def layout(self):
     def requirements(self):
         self.requires(self.tested_reference_str)
 
+    def generate(self):
+        zstd_version = self.dependencies["zstd"].ref.version
+        save(self, "zstd_version", f"{zstd_version}")
+
+
     def build(self):
         cmake = CMake(self)
         cmake.configure()
         cmake.build()
 
     def test(self):
-        if can_run(self):
```

Comment:
The original test must be preserved. It validates the packaged library. Please, revert it.